docs/source/user/cfunc.rst

```diff
@@ -216,3 +216,44 @@ Compilation options
 A number of keyword-only arguments can be passed to the ``@cfunc``
 decorator: ``nopython`` and ``cache``.  Their meaning is similar to those
 in the ``@jit`` decorator.
+
+
+Calling C code from Numba
+=========================
+
+It is also possible to call C code from Numba ``@jit`` functions. In this
+example, we are going to be compiling a simple function ``sum`` that adds two
+integers and calling it within Numba ``@jit`` code
+
+.. code-block:: C
```

Comment:
```suggestion
.. code-block:: C

```

I think this is the cause of the doc build error here:

```
Warning, treated as error:
/home/vsts/work/1/s/docs/source/user/cfunc.rst:228:Error in "code-block" directive:
maximum 1 argument(s) allowed, 3 supplied.

.. code-block:: C
   #include <stdint.h>

   int64_t sum(int64_t a, int64_t b){
      return a + b;
   }
Makefile:53: recipe for target 'html' failed
```

Reply:
Hi @esc, just fixed the issue.